### PR TITLE
Platform_machine names for greenlet check

### PR DESCRIPTION
**Pull request by jdow7**
### Description

Correct arch64 to aarch64 and and all caps AMD64 to platform machine list for greenlet dependency in setup.cfg
Issue #7024 

### Checklist

This pull request is:

- [ ] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [x] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision dbd7419e21d2eb96a02b34a7f41de8a0382be559 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change dbd7419e21d2eb96a02b34a7f41de8a0382be559: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3094

File: setup.cfg [hunk 1]
```diff
@@ -40,7 +40,7 @@ package_dir =
 
 install_requires =
     importlib-metadata;python_version<"3.8"
-    greenlet != 0.4.17;python_version>='3' and (platform_machine=='arch64' or (platform_machine=='ppc64le' or (platform_machine=='x86_64' or (platform_machine=='amd64' or (platform_machine=='win32' or platform_machine=='WIN32')))))
+    greenlet != 0.4.17;python_version>='3' and (platform_machine=='aarch64' or (platform_machine=='ppc64le' or (platform_machine=='x86_64' or (platform_machine=='amd64' or (platform_machine=='AMD64' or (platform_machine=='win32' or platform_machine=='WIN32'))))))
```
Comment (CaselIT) [inline setup.cfg:43]: we are almost at the point where it may be better to script this -.-'

they could have devised a better way of doing this...

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3094 has been **merged**. Congratulations! :)

**Comment by zzzeek** [comment]
thanks very much for this!

**Pull request closed without merging**